src/gui.ml

```diff
@@ -168,3 +168,21 @@ let init_gui () =
   set_window_title "OCaml Chess";
   set_line_width 2;
   imgs := load_imgs ()
+  
+let rec draw_win_screen (result :color)= 
+  set_color white;
+  fill_rect 75 250 450 100;
+  set_color black;
+  set_font "-*-fixed-medium-r-semicondensed--20-*-*-*-*-*-iso8859-1";
+  moveto 100 300;
```

Comment:
Should have fill_rect and moveto be relative to the window length and step defined in this file. Will allow us to scale the board if we ever get to that in the future.